pkg/services/alerting/notifiers/slack.go

```diff
@@ -24,25 +28,29 @@ func init() {
 	alerting.RegisterNotifier(&alerting.NotifierPlugin{
 		Type:        "slack",
 		Name:        "Slack",
-		Description: "Sends notifications to Slack via Slack Webhooks",
+		Description: "Sends notifications to Slack",
 		Heading:     "Slack settings",
 		Factory:     NewSlackNotifier,
 		Options: []alerting.NotifierOption{
 			{
-				Label:        "Url",
+				Label:        "Recipient",
```

Comment:
Unfortunately making this required is a bit weird if you're using the webhook url. See below for one of my already configured channels that I tried to update:
![image](https://user-images.githubusercontent.com/1668778/115693476-3ff2d980-a360-11eb-8e52-d2c525c0a720.png)

Reply:
@marefr thanks for catching this, I've changed it so the recipient field is not required, but you get an error if you try to save the notifier without setting one unless a webhook is provided.